src/backend/joanie/core/serializers.py

```diff
@@ -320,92 +319,32 @@ class CourseSerializer(serializers.ModelSerializer):
     """
 
     organizations = OrganizationSerializer(many=True, read_only=True)
```

Comment:
do you really need organizations? This is not the "selling" organizations but the "authoring" organizations right?

Reply:
With @NathanVss, we wonder which thumbnail to display in dashboard course list view for "standalone" course runs : 
- Should we add a field thumbnail to the course ?
- Should we display organization logo is there is only organization and the "main" organization in other case ?

Reply:
> Gravé dans la roche 🎶
https://github.com/openfun/joanie/issues/201